UPDATING.md

```diff
@@ -26,6 +26,7 @@ assists people when migrating to a new version.
 
 ### Breaking Changes
 
+- [18976](https://github.com/apache/superset/pull/18976): When running the app in debug mode, the app will default to use `SimpleCache` for `FILTER_STATE_CACHE_CONFIG` and `EXPLORE_FORM_DATA_CACHE_CONFIG`. When running in non-debug mode, a cache backend will need to be defined, otherwise the application will fail to start. For installations using Redis or other caching backends, it is recommended to use the same backend for both cache configs.
```

Comment:
> For installations using Redis or other caching backends, it is recommended to use the same backend for both cache configs.

I'm wondering if there is a real need in configuring different cache expiration for both cache storages? If not, maybe we should just merge them (albeit it'd be a bigger breaking change)?